frontend/src/components/accountPage/UserPanel/UserPanel.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from 'react';
+import { Row, Col, Avatar, Tabs, Table } from 'antd';
+import { UserOutlined } from '@ant-design/icons';
+import UserInfo from '../UserInfo/UserInfo';
+
+const { TabPane } = Tabs;
+const { Column } = Table;
+export interface IUserPanelProps {
+  firstName: string;
+  lastName: string;
+  username: string;
+  email: string;
+  avatar?: string;
+  sshKeys?: { name: string; key: string }[];
+}
+
+const UserPanel: FC<IUserPanelProps> = props => {
+  const { avatar, sshKeys, ...otherInfo } = props;
+
+  return (
+    <Row className="p-4">
```

Comment:
In the updated version we added that attribute but the problem with vertical alignment is that whenever the container's height changes, the content's vertical postition is updated. The container should either have a fixed height or something like that to prevent similar graphical glitches